### PR TITLE
re-number tutorials

### PR DESCRIPTION
**Pull request by alemata**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you review?

File: articles/client-platforms/angular2/07-authorization.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: This tutorial will show you how assign roles to your users, and use
   link: 'https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/07-Authorization',
 }) %>
 
-<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/angular2/07-rules' }) %>
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/angular2/06-rules' }) %>
```
Comment (flaviogranero) [inline articles/client-platforms/angular2/07-authorization.md:10]: shouldn't the file name here be 06-rules.md?

Comment (alemata) [inline articles/client-platforms/angular2/07-authorization.md:10]: It's used to point to the rules tutorial [here](https://github.com/auth0/docs/blob/master/articles/client-platforms/_includes/_authorization-introduction.md), so it's ok without md extension...

**Comment by flaviogranero** [comment]
👍 @ntotten

**Pull request merged**